dash/dash.py

```diff
@@ -340,6 +336,7 @@ def _config(self):
             'url_base_pathname': self.url_base_pathname,
             'requests_pathname_prefix': self.config.requests_pathname_prefix,
             'dev_tools_ui': self._dev_tools.dev_tools_ui,
+            'dev_tools_props_check': self._dev_tools.dev_tools_props_check,
```

Comment:
with this, the props-check will only do when `bool(dev_tools_ui && dev_tools_props_check) == True`